lib/main.dart

```diff
@@ -45,189 +47,209 @@ class _MyHomePageState extends State<MyHomePage> {
+                              decoration: InputDecoration(
+                                fillColor:const  Color.fromRGBO(0, 0, 0, 0.3),
+                                filled: true,
+                                focusedBorder: OutlineInputBorder(
+                                  borderRadius: BorderRadius.circular(20),
+                                  borderSide:const  BorderSide(color: Colors.white),
+                                ),
+                                hintText: "Password",
+                                hintStyle:const  TextStyle(color: Colors.white),
+                                border: OutlineInputBorder(
+                                  borderRadius: BorderRadius.circular(20),
+                                ),
+                                prefixIcon: const Icon(
+                                  Icons.lock_outline,
                                   color: Colors.white,
                                 ),
-                          onPressed: vaw,
-                        ),
+                                suffixIcon: IconButton(
+                                  icon: eye == true
+                                      ? const Icon(
+                                          Icons.remove_red_eye_outlined,
+                                          color: Colors.white,
+                                        )
+                                      : const Icon(
+                                          Icons.remove_red_eye,
+                                          color: Colors.white,
+                                        ),
+                                  onPressed: vaw,
+                                ),
+                              ),
+                            ),
+                          ),
+                          const SizedBox(
+                            height: 20,
+                          ),
+                          MaterialButton(
+                            color:const Color.fromRGBO(0, 0, 0, 0.3),
+                            minWidth: MediaQuery.of(context).size.width * .8,
+                            height: 55,
+                            shape: const RoundedRectangleBorder(
+                                borderRadius: BorderRadius.all(Radius.circular(20.0))),
+                            onPressed: () {},
+                            child:const Text(
+                              "Sign Up",
+                              style: TextStyle(color: Colors.white),
+                            ),
+                          ),
+                          const SizedBox(
+                            height: 40,
+                          ),
+                          Row(
+                            mainAxisAlignment: MainAxisAlignment.center,
+                            crossAxisAlignment: CrossAxisAlignment.end,
+                            children: [
+                              FloatingActionButton(
+                                heroTag: '6',
```

Comment:
What's the point to use Hero animation if you haven't written the onPress action.